docs/setup/memfootprint-test/memfootprint-test-setup.md

```diff
@@ -0,0 +1,75 @@
+# Measuring memory footprint of EdgeCore
+
+## Why measuring memory footprint
+
+- This platform is also tagged for a light weighted edge computing deployment
+- To be able to be deployed over devices with less resources (for example, 256MB RAM)
+- It is required to know by deploying as many as possible pods, it showcases as much as less possible memory footprint
+
+## KPI's measured
+
+- %CPU
+- %Memory
+- Resident Set Size (RSS)
+
+## How to test
+
+After deployment and provisioning of kubeedge cloud and edge components in 2 VM's (supported and tested over ubuntu 16.04) respectively, start deploying pods from 0 to 100 in steps of 5. Keep capturing above KPI's using standard linux "ps" commands, after each step.
+
+### Test setup
+
+![Kubeedge TestSetup](./perftestsetup_diagram.PNG)  
+
+*Fig 1: Kubeedge TestSetup*
+
+### Creating a setup
+
+#### Requirements
+
+- Host machine's or VM's resource requirements can mirror the edge device of your choice
+- Resources used for above setup are 4 CPU, 8GB RAM and 200 GB Disk space. OS is Ubuntu 16.04.
+- Docker image used to deploy the pods in edge, needs to be created. The steps are:
+    1. Go to github.com/kubeedge/kubeedge/docs/setup/memfootprint-test/
```

Comment:
Please do not place your dockerfiles in the docs folder. It can only be used for documentation.